functional/vfio/run.sh

```diff
@@ -214,9 +227,12 @@ main() {
 	addr=$(get_eth_addr)
 	[ -n "${addr}" ] || die "virtio ethernet controller address not found"
 
-	unbind_pci_dev "${addr}"
+	# it is possible the device is not binded, do not fail
+	unbind_pci_dev "${addr}" || true
```

Comment:
My preference is for changing the `unbind_pci_dev` to not attempt to unbind the not binded device. This way we avoid noise on the logs, just like:

```
+ '[' -n 06:00.0 ']'
+ unbind_pci_dev 06:00.0
+ addr=06:00.0
+ echo 0000:06:00.0
+ sudo tee /sys/bus/pci/devices/0000:06:00.0/driver/unbind
tee: '/sys/bus/pci/devices/0000:06:00.0/driver/unbind': No such file or directory
0000:06:00.0
+ true
```

Likewise for `unbind_from_vfio`

What's your opinion @marcel-apf ?

Reply:
@wainersm  we need that in order to be able to run the vfio test more than once on a vm. In order to solve the latest SRIOV PR issues I needed to run the test at least 100 times... I hardly imagine how time would it take if I needed to run the whole setup again and again.
That being said this kind of errors are not nice - however the test is far for being perfect and I concentrated on the task in hand...

Reply:
I just read again your comment, I'll see how can I detect nicely if the device is binded already or not (instead of the ugly  `|| true`)

Reply:
@marcel-apf great :)

Reply:
Or.. if you use `driverctl` it will handle that for you :).